### PR TITLE
docker_swarm inventory plugin: add basic integration tests

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
As topic says.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
lib/ansible/plugins/inventory/docker_swarm.py

### CODE REVIEW

**Comment by felixfontein** [comment]
bot_status

**Comment by felixfontein** [comment]
@WojciechowskiPiotr @morph027 looks like ansibot doesn't really associate it to the inventory plugin right now.

**Comment by felixfontein** [comment]
ansible-test does pick it up, though:
```
$ test/runner/ansible-test integration --changed-path lib/ansible/plugins/inventory/docker_swarm.py -e
lib/ansible/plugins/inventory/docker_swarm.py -> integration: inventory_docker_swarm (focused) (targeted)
$ test/runner/ansible-test integration --changed-path test/integration/targets/inventory_docker_swarm/runme.sh -e
test/integration/targets/inventory_docker_swarm/runme.sh -> integration: inventory_docker_swarm (focused) (targeted)
```
I guess ansibot will notice once this is merged...

File: test/integration/targets/inventory_docker_swarm/aliases [hunk 1]
```diff
@@ -0,0 +1,10 @@
+shippable/posix/group2
+skip/osx
+skip/freebsd
+destructive
+skip/docker  # The tests sometimes make docker daemon unstable; hence,
```
Comment (gundalow) [inline test/integration/targets/inventory_docker_swarm/aliases:5]: Thanks for adding these details.

File: test/integration/targets/inventory_docker_swarm/playbooks/test_inventory_1.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+- hosts: all
+  connection: local  # otherwise Ansible will complain that it cannot connect via ssh to 127.0.0.1:22
```
Comment (gundalow) [inline test/integration/targets/inventory_docker_swarm/playbooks/test_inventory_1.yml:3]: Yay for good comments.

**Review by gundalow — APPROVED** [review]

**Comment by felixfontein** [comment]
@gundalow thanks for merging!

**Pull request merged**